### PR TITLE
Close #48 html for categories

### LINKED ISSUE #48: HTML for categories
As a user, I can view categories

### PR DESCRIPTION
**Pull request by Jpease1020**
Closing

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -19,8 +19,9 @@ def create
 
   def show
     @user = current_user
+    byebug
```
Comment (emilyfreeman) [inline app/controllers/users_controller.rb]: this byebug needs to be removed

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<center><h2>Categories</h2></center>
+
+<div id="categories">
+  <div class="row">
+    <% @categories.each do |category| %>
+    <div class="col m4" id="grayscale">
+      <div class="card" id="<%=category.slug%>">
```
Comment (emilyfreeman) [inline app/views/categories/index.html.erb]: It's nitpicky, but I'd like to see a space on either side within the erb tag

**Comment by emilyfreeman** [comment]
Just a few little things to update before merging

**Comment by slota** [comment]
Made necessary changes suggested by Emily, merging

**Pull request merged**